### DISCUSSION TITLE
Making installed packages available to the current session?

### DISCUSSION THREAD

**Discussion by wbehrens-on-gh** [microsoft/winget-cli#2614] [Q&A] [answered]
if I run `winget install EclipseAdoptium.Temurin.11.JDK` to install the jdk in order for the `java` command to be available to me I have to open a new cmd session. Is there a way to make it available to the current session to be used in a script?

**Comment by denelon** [comment, marked as answer]
It's likely that the installer is adding the appropriate environment variables, but they aren't ready for use without restarting the current shell session or reloading the environment variables.

https://stackoverflow.com/questions/17794507/reload-the-path-in-powershell
